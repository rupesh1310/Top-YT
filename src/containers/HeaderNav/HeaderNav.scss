.ui.menu.top-menu {
  border: none;
  .logo {
    width: 17rem;
  }
  .nav-container {
    flex-grow: 1;
    padding: 0;

    .search-input {
      padding-left: 0;
      width: 33%;

      form {
        width: 100%;
      }
    }
  }

  .header-icon {
    color: #a0a0a0;
  }
}