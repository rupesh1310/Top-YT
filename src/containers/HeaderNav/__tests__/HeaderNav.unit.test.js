import React from 'react';
import {shallow} from 'enzyme';
import HeaderNav from '../HeaderNav';

describe('HeaderNav', () => {
  test('renders', () => {
    const wrapper = shallow(
      <HeaderNav />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
