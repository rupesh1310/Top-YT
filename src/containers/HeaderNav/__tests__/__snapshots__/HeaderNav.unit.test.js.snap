// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav renders 1`] = `
<Route>
  <Component />
</Route>
`;
